packages/http-client/src/ceramic-http-client.ts

```diff
@@ -125,6 +125,12 @@ export default class CeramicClient implements CeramicApi {
     const docIdStr = doc.id.toString()
     if (!this._docmap[docIdStr]) {
       this._docmap[docIdStr] = doc
+    } else if (
+      JSON.stringify(DoctypeUtils.serializeState(doc.state)) !==
```

Comment:
can we move this comparison to `DoctypeUtils` as well? Something like `compareStates` or similar

Reply:
Fixed :)

Reply:
🤘